### DISCUSSION TITLE
Firefox 2FA don't work with Ente Auth

### DISCUSSION THREAD

**Discussion by Pato98** [ente-io/ente#3970] [Q&A] [answered]
I tried to add 2fa to Ente Auth manually and with the QR code, but it didn't worked.
Invalid 2fa code.
With Authy it's working.
I tried to add 2fa on accounts.firefox.com.

**Comment by xtbqimuxmvtx** [comment, marked as answer]
Works fine for me - 6 digit 30 second codes and my secret is 32 characters

Double check your system time?

If you add it to both Authy and Ente you should see the same code in both

**Reply by Pato98** [reply]
Thanks for your help. Now I tried it from my phone browser and it's worked I added it to Ente Auth.
